{"name": "Login with valid credentials -- @1.3 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"link text\",\"selector\":\"Edit your account information\"}\n  (Session info: chrome=128.0.6613.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF692169632+30946]\n\t(No symbol) [0x00007FF69211E3C9]\n\t(No symbol) [0x00007FF692016FDA]\n\t(No symbol) [0x00007FF69206822C]\n\t(No symbol) [0x00007FF69206850C]\n\t(No symbol) [0x00007FF6920ADCB7]\n\t(No symbol) [0x00007FF69208CAAF]\n\t(No symbol) [0x00007FF6920AB041]\n\t(No symbol) [0x00007FF69208C813]\n\t(No symbol) [0x00007FF69205A6E5]\n\t(No symbol) [0x00007FF69205B021]\n\tGetHandleVerifier [0x00007FF69229F83D+1301229]\n\tGetHandleVerifier [0x00007FF6922ABDB7+1351783]\n\tGetHandleVerifier [0x00007FF6922A2A03+1313971]\n\tGetHandleVerifier [0x00007FF69219DD06+245686]\n\t(No symbol) [0x00007FF69212758F]\n\t(No symbol) [0x00007FF692123804]\n\t(No symbol) [0x00007FF692123992]\n\t(No symbol) [0x00007FF69211A3EF]\n\tBaseThreadInitThunk [0x00007FFDD3BA7374+20]\n\tRtlUserThreadStart [0x00007FFDD569CC91+33]\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login.py\", line 30, in step_impl\n    assert context.account_page.check_account_enter()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\features\\pages\\accountpage.py\", line 12, in check_account_enter\n    return self.driver.find_element(By.LINK_TEXT, self.confirm_text).is_displayed()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 748, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Login page", "status": "passed", "start": 1725724932343, "stop": 1725724932813}, {"name": "When I enter valid email as 2136823@gmail.com and valid password as 123456789 into fields", "status": "passed", "start": 1725724932813, "stop": 1725724933083}, {"name": "And I click on Login button", "status": "passed", "start": 1725724933083, "stop": 1725724933714}, {"name": "Then I get logged", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"link text\",\"selector\":\"Edit your account information\"}\n  (Session info: chrome=128.0.6613.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF692169632+30946]\n\t(No symbol) [0x00007FF69211E3C9]\n\t(No symbol) [0x00007FF692016FDA]\n\t(No symbol) [0x00007FF69206822C]\n\t(No symbol) [0x00007FF69206850C]\n\t(No symbol) [0x00007FF6920ADCB7]\n\t(No symbol) [0x00007FF69208CAAF]\n\t(No symbol) [0x00007FF6920AB041]\n\t(No symbol) [0x00007FF69208C813]\n\t(No symbol) [0x00007FF69205A6E5]\n\t(No symbol) [0x00007FF69205B021]\n\tGetHandleVerifier [0x00007FF69229F83D+1301229]\n\tGetHandleVerifier [0x00007FF6922ABDB7+1351783]\n\tGetHandleVerifier [0x00007FF6922A2A03+1313971]\n\tGetHandleVerifier [0x00007FF69219DD06+245686]\n\t(No symbol) [0x00007FF69212758F]\n\t(No symbol) [0x00007FF692123804]\n\t(No symbol) [0x00007FF692123992]\n\t(No symbol) [0x00007FF69211A3EF]\n\tBaseThreadInitThunk [0x00007FFDD3BA7374+20]\n\tRtlUserThreadStart [0x00007FFDD569CC91+33]\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login.py\", line 30, in step_impl\n    assert context.account_page.check_account_enter()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\features\\pages\\accountpage.py\", line 12, in check_account_enter\n    return self.driver.find_element(By.LINK_TEXT, self.confirm_text).is_displayed()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 748, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProject12\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "screenshot", "source": "67f33666-88f1-4f4b-9c56-d20ceae7a19e-attachment.png", "type": "image/png"}], "start": 1725724933714, "stop": 1725724933913}], "parameters": [{"name": "email", "value": "2136823@gmail.com"}, {}, {"name": "password", "value": "123456789"}], "start": 1725724926605, "stop": 1725724936074, "uuid": "64adb866-8297-47dd-aa02-dcb8a4f16203", "historyId": "95b04651546eaf5e304293921e76549f", "fullName": "Login Functionality: Login with valid credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "login"}, {"name": "feature", "value": "Login Functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}